### PR TITLE
Updates for post-0.5.1 of Antidote

### PR DESCRIPTION
**Pull request by Mierdin**
Antidote 0.5.0 introduced a lot of changes to the overall architecture of Antidote, specifically the front-end. A big piece of this was the removal of Guacamole in favor of a more streamlined solution involving xtermjs and a socket.io/ssh proxy called webssh2.

Immediately after these changes went live on the NRE Labs site, I got to work re-vamping the back-end services (formerly Syringe), so I didn't have time to invest in updating selfmedicate. These efforts are going into version 0.6.0 of the Antidote platform.

Now that the bulk of these efforts are done, I wanted to spend some time finally finishing the updates to selfmedicate to be compatible. This PR removes all the old stuff and adds the new stuff like webssh2 and should be compatible with the latest version of the Antidote platform. Note that this PR uses the `latest` images for all antidote components. Once 0.6.0 is actually released, I may follow up with a PR that locks into this version for some additional stability.

> ALSO NOTE that this PR was originally spun up as a very early PoC for webssh functionality, and most of the conversation below from late last year was a result of this. The PR description here was changed to reflect the overall and final goal of the PR, which is to simply update the whole of selfmedicate to be compatible with the latest platform developments.

### CODE REVIEW

**Comment by nlundquist** [comment]
Hmm I've spun up this branch and I'm not able to access the webssh instance. I've tried URLs like: 

- https://antidote-local/ssh/host/10.103.81.189?port=22
- http://antidote-local:30010/ssh/host/10.103.81.189?port=22

 but I just get a connection refused error.

**Comment by Mierdin** [comment]
I'll bet I forgot to add the manifest to the script. Can you run "kubectl
create -f manifests/webssh2.yaml" in the vagrant machine?

**Comment by nlundquist** [comment]
```
vagrant@antidote-040:~$ kubectl create -f manifests/webssh2.yaml
namespace/test created
deployment.extensions/webssh2 created
deployment.extensions/utility created
service/webssh2 created
service/utility created
ingress.extensions/webssh2 created
```

...however I'm still not seeing a change in behavior - do I need to halt and resume the machine?

What port should the webssh server be running on?

**Comment by Mierdin** [comment]
Wow. I did something very, very wrong here. The file in this PR is not in the state I intended at all - must have added/committed in a hurry. Let me try again - give me a few minutes.

**Comment by Mierdin** [comment]
@nlundquist Okay......everything **should** be okay now. I tested from scratch twice to be sure. Let me know if you run into more problems.
